apps/www/registry/stories/accordion.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/registry/default/ui/accordion"
+
+/**
+ * A vertically stacked set of interactive headings that each reveal a section
+ * of content.
+ */
+const meta: Meta<typeof Accordion> = {
+  title: "ui/Accordion",
+  component: Accordion,
+  tags: ["autodocs"],
+  argTypes: {},
+}
+
+export default meta
+
+type Story = StoryObj<typeof Accordion>
```

Comment:
Given that you're using TS 4.9+, I recommend ([here's why](https://storybook.js.org/blog/improved-type-safety-in-storybook-7/)) typing your CSF files like this:

```suggestion
const meta = {
  title: "ui/Accordion",
  component: Accordion,
  tags: ["autodocs"],
  argTypes: {},
} satisfies Meta<typeof Accordion>;

export default meta

type Story = StoryObj<typeof meta>
```

Then each story would be typed like this:

```ts
export const Default = {
  // ...
} satisfies Story;

Reply:
very nice, i'll try the codemod *`npx storybook migrate upgrade-deprecated-types --glob="**/*.stories.@(js|jsx|ts|tsx)"` 👍